### DISCUSSION TITLE
Portfolio website

### DISCUSSION THREAD

**Discussion by pk-rcb** [smaranjitghose/awesome-portfolio-websites#635] [Ideas]
Hello sir i am pratik khuntia i have costomised your codes and made it for me thanks for providing this 
I am a beginner i wanna learn things frm you plz help me . See my site i have created[My site](https://pk-rcb.github.io/pratik.github.io/index.html)

**Comment by zakarialaoui10** [comment]
Nice

**Reply by pk-rcb** [reply]
Ys
